webui/react/src/components/TaskCard.stories.tsx

```diff
@@ -22,6 +22,7 @@ const baseProps: RecentTask = {
   },
   ownerId: 5,
   progress: 0.34,
+  startTime: (new Date()).toString(),
```

Comment:
suggestion: use `Date.now()` instead of `new Date()`
https://stackoverflow.com/questions/12517359/performance-date-now-vs-date-gettime